include/os/freebsd/spl/sys/debug.h

```diff
@@ -178,16 +289,15 @@ spl_assert(const char *buf, const char *file, const char *func, int line)
 #define	ASSERT3P	VERIFY3P
 #define	ASSERT0		VERIFY0
 #define	ASSERT0P	VERIFY0P
+#define	ASSERT3Bf	VERIFY3Bf
```

Comment:
Could we also get VERIFYf/ASSERTf?

Reply:
There, now it exists and even compiles if used.